src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The draw() method is not conceptually close to the getArea() method. According to the checklist, don't put all behavior into a single interface if the methods are conceptually different from each other. All our classes and interfaces should have a single purpose.